var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -57,5 +63,12 @@ class Mesa(AutotoolsPackage):
     depends_on('presentproto@1.0:', type='build')
     depends_on('pkg-config@0.9.0:', type='build')
 
+
+    def configure_args(self):
+        make('autoreconf -fi')
```

Comment:
It looks like the newer version fixes this autoreconf problem (it looks like spack actually somehow calls autoreconf automatically). I fixed this issue in 61d8cebbe097edafa82c2c2bb0e8e6578022707, but I hit a problem that now I need to comment out the older version otherwise the url somehow fails to download it. So the way we implemented the `url` in `version()` above I think is incorrect.

Anyway, for now I am commenting out, and continuing on the quest to build paraview.